detection-rules/body_business_email_compromise_new_sender.yml

```diff
@@ -23,7 +23,7 @@ source: |
           )
   )
 tags:
-  - "Machine Learning"
+  - "Machine learning"
   - "Natural Language Understanding"
   - "Suspicious sender"
-  - "Business Email Compromise"
+  - "Business email compromise"
```

Comment:
@itsojon i think we want this want to be capitalized cuz it's a known acronym (BEC)